mnelab/dialogs/__init__.py

```diff
@@ -2,27 +2,31 @@
 #
 # License: BSD (3-clause)
 
-from .annotations import AnnotationsDialog
-from .append import AppendDialog
+from .append_data import AppendDataDialog
 from .calc import CalcDialog
-from .channel_properties import ChannelPropertiesDialog
-from .crop import CropDialog
+from .change_reference import ChangeReferenceDialog
+from .create_epochs import CreateEpochsDialog
+from .crop_data import CropDataDialog
 from .drop_bad_epochs import DropBadEpochsDialog
-from .epoch import EpochDialog
-from .erds import ERDSDialog, ERDSTopomapsDialog
+from .edit_annotations import EditAnnotationsDialog
+from .edit_channel_properties import EditChannelPropertiesDialog
+from .edit_events import EditEventsDialog
 from .error_message import ErrorMessageBox
-from .events import EventsDialog
-from .filter import FilterDialog
+from .filter_data import FilterDataDialog
 from .find_events import FindEventsDialog
 from .history import HistoryDialog
-from .interpolate_bads import InterpolateBadsDialog
-from .mat import MatDialog
-from .meta_info import MetaInfoDialog
-from .montage import MontageDialog
+from .interpolate_bad_channels import InterpolateBadChannelsDialog
+from .mat_variables import MatVariablesDialog
 from .pick_channels import PickChannelsDialog
-from .plot_evoked import PlotEvokedComparisonDialog, PlotEvokedDialog, PlotEvokedTopomaps
-from .reference import ReferenceDialog
+from .plot_erds import PlotERDSMapsDialog, PlotERDSTopomapsDialog
+from .plot_evoked import (
+    PlotEvokedComparisonDialog,
+    PlotEvokedDialog,
+    PlotEvokedTopomapsDialog,
+)
 from .rename_channels import RenameChannelsDialog
 from .run_ica import RunICADialog
-from .xdf_chunks import XDFChunksDialog
+from .set_montage import SetMontageDialog
+from .show_xdf_chunks import ShowXDFChunksDialog
+from .show_xdf_meta_information import ShowXDFMetaInformationDialog
```

Comment:
This is getting too long – can we call it `ShowXDFInformationDialog` everywhere?